### PR TITLE
[14.0][FIX] sale_invoice_plan: product qtd round precision - Ref. 2181

### PR DESCRIPTION
**Pull request by marcelsavegnago**
Port from #2181

### CODE REVIEW

**Review by rvalyi — APPROVED** [review]

File: sale_invoice_plan/models/sale_invoice_plan.py [hunk 1]
```diff
@@ -112,6 +112,7 @@ def _compute_new_invoice_quantity(self, invoice_move):
             else:
                 plan_qty = order_line.product_uom_qty * (percent / 100)
                 prec = order_line.product_uom.rounding
+                plan_qty = round(plan_qty, precision_rounding=prec)
```
Comment (ivantodorovich) [inline sale_invoice_plan/models/sale_invoice_plan.py]:
```suggestion
                plan_qty = float_round(plan_qty, precision_rounding=prec)
```

and don't forget to import it.

this is why CI is red

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

**Review by kittiu — APPROVED** [review]

**Review by ivantodorovich — APPROVED** [review]

File: sale_invoice_plan/models/sale_invoice_plan.py (hunk 1 above)
Comment (marcelsavegnago) [reply sale_invoice_plan/models/sale_invoice_plan.py]: done

**Comment by marcelsavegnago** [comment]
ping

**Pull request merged**